types/qrcode/index.d.ts

```diff
@@ -164,10 +164,18 @@ export function toDataURL(canvasElement: HTMLCanvasElement, text: string | QRCod
  * Returns a Data URI containing a representation of the QR Code image.
  */
 export function toDataURL(text: string | QRCodeSegment[], callback: (error: Error, url: string) => void): void;
+/**
```

Comment:
You would want the signature of toDataUrl that returns promise before the existing signature since the new one takes fewer parameters